packages/react-native-reanimated/.lintstagedrc.js

```diff
@@ -1,14 +1,5 @@
-const REANIMATED_PATH = 'packages/react-native-reanimated';
+const commonConfig = require('../../.lintstagedrc-common.js');
+
 module.exports = {
-  '*.(js|ts|tsx)': [
-    `cd ${REANIMATED_PATH} && yarn eslint `,
-    `cd ${REANIMATED_PATH} && yarn eslint --quiet --ext '.js,.ts,.tsx' src/`,
-    `cd ${REANIMATED_PATH} && yarn prettier --write`,
-  ],
-  'plugin/**/*.{js,ts,tsx}': `cd ${REANIMATED_PATH} && yarn lint:plugin`,
-  '**/*.{h,cpp}': `cd ${REANIMATED_PATH} && yarn lint:cpp`,
-  'android/src/**/*.java': `cd ${REANIMATED_PATH} && yarn format:java`,
-  'android/src/**/*.{h,cpp}': `cd ${REANIMATED_PATH} && yarn format:android`,
-  'apple/**/*.{h,m,mm,cpp}': `cd ${REANIMATED_PATH} && yarn format:ios`,
-  'Common/**/*.{h,cpp}': `cd ${REANIMATED_PATH} && yarn format:common`,
```

Comment:
Turns out these complex commands don't always work and this syntax is unreliable in `lint-staged`. Read more about it in `.husky/pre-commit`.